bot/exts/pride/prideleader.py

```diff
@@ -0,0 +1,87 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Colours
+log = logging.getLogger(__name__)
+
+
+class PrideLeader(commands.Cog):
+    """Gives a Pride Leader Info."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.pride = self.load_json()
+
+    @staticmethod
+    def load_json() -> dict:
+        """Loads pride leader information from static json resource."""
+        explanation_file = Path("bot/resources/pride/prideleader.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            pride = json.load(json_data)
+
+        return pride
+
+    def name_verifier(self, leader_name: str) -> str:
+        """Verify leader name wether it is present in json or not."""
+        leader_name = leader_name.split(" ")
+        leader_name = ' '.join(name.capitalize() for name in leader_name)
+        if leader_name in self.pride:
+            log.trace("Got Valid name.")
+            return leader_name
+
+    def invalid_embed_generate(self, pride_leader: str) -> discord.Embed:
+        """Genrates Invalid Embed."""
+        embed = discord.Embed()
+        embed.color = Colours.soft_red
+        valid_name = []
+        pride_leader = pride_leader.split(" ")
+        pride_leader = ' '.join(name.capitalize() for name in pride_leader)
+        for name in self.pride.keys():
+            if fuzz.ratio(pride_leader, name) >= 40:
+                valid_name.append(name)
+        if len(valid_name) == 0:
+            valid_name = ", ".join([name for name in self.pride.keys()])
```

Comment:
```suggestion
            valid_names = ", ".join(self.pride)
```
Multiple names, plural variable name (remember to change the rest)